notifications.bs

```diff
@@ -294,13 +294,15 @@ removed from the <a>list of notifications</a>.
 <h3 id="permissions-integration">Permissions integration</h3>
 
 <p>The Notifications API is a <a>powerful feature</a> which is identified by the
-<a for="powerful feature">name</a> "{{PermissionName/notifications}}". [[!Permissions]]
+<a for="powerful feature">name</a>
+"<dfn export permission id="notification-permission-name"><code>notifications</code></dfn>".
```

Comment:
Apologies, final question. Can the ID be removed or was it there to preserve something?
```suggestion
<a for="powerful feature">name</a> "<dfn export permission><code>notifications</code></dfn>".
```

Reply:
Oh - yeah, it can be removed. I hadn't updated my version of bikeshed, and it was attempting to link to `#concept-notification` - it now links to `#permissiondef-notifications`.